src/tests/ftest/recovery/container_cleanup.py

```diff
@@ -0,0 +1,129 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from recovery_test_base import RecoveryTestBase
+from general_utils import report_errors
+from exception_utils import CommandFailure
+
+
+class ContainerCleanupTest(RecoveryTestBase):
+    """Test Pass 5: Container Cleanup
+
+    :avocado: recursive
+    """
+
+    def test_container_label_inconsistency(self):
+        """Test container label inconsistency in CS and property.
+
+        1. Create a pool and a container.
+        2. Inject fault to cause container label inconsistency. i.e., Corrupt label in CS.
+        3. Try to access property using the original label in CS.
+        4. Show that PS has the original label by using new-label.
+        5. Enable the checker.
+        6. Set policy to --all-interactive.
+        7. Start the checker and query the checker until the fault is detected.
+        8. Repair by selecting "Trust the container label in container property."
+        9. Query the checker until the fault is repaired.
+        10. Disable the checker.
+        11. Verify that the inconsistency was fixed.
+
+        Jira ID: DAOS-12289
+
+        :avocado: tags=all,pr
+        :avocado: tags=vm
+        :avocado: tags=recovery,container_cleanup
+        :avocado: tags=ContainerCleanupTest,test_container_label_inconsistency
+        """
+        # 1. Create a pool and a container.
+        self.log_step("Create a pool and a container")
+        pool = self.get_pool(connect=False)
+        container = self.get_container(pool=pool)
+
+        # 2. Inject fault to cause container label inconsistency.
+        self.log_step("Inject fault to cause container label inconsistency.")
+        daos_command = self.get_daos_command()
+        daos_command.faults_container(
+            pool=pool.identifier, cont=container.identifier,
+            location="DAOS_CHK_CONT_BAD_LABEL")
+
+        # 3. Try to access property using the original label in CS.
+        self.log_step("Try to access property using the original label in CS.")
+        access_failed = False
+        try:
+            _ = daos_command.container_get_prop(
+                pool=pool.identifier, cont=container.identifier)
+        except CommandFailure:
+            access_failed = True
+        if not access_failed:
+            self.fail("Label inconsistency fault wasn't injected property!")
```

Comment:
Python has this nice syntax
```suggestion
        try:
            _ = daos_command.container_get_prop(
                pool=pool.identifier, cont=container.identifier)
        except CommandFailure:
            pass
        else:
            self.fail("Label inconsistency fault wasn't injected property!")
```

Where the `else` runs if `CommandFailure` wasn't raised

Reply:
Done. Thanks.